src/engine/Settings.cs

```diff
@@ -101,6 +101,7 @@ public static Settings Instance
     ///   Sets amount of MSAA to apply to the viewport
     /// </summary>
     public Viewport.MSAA MSAAResolution { get; set; } = Viewport.MSAA.Disabled;
+    public int ColorblindSetting { get; set; } = 0;
```

Comment:
add a blank line before this, and add a documentation comment on this, especially documenting the possible values as this isn't an enum. Also rename to `ColourblindSetting` as unlike Godot we know how to spell colour.